net.runelite.RuneLite.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "net.runelite.RuneLite",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "runelite",
+    "separate-locales": false,
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10" 
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--env=JAVA_HOME=/app/jre",
+        "--persist=~"
+    ],
+    "modules": [
+        {
+            "name": "openjdk",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/openjdk10/install.sh"
+            ]
+        },
+        {
+            "name": "runelite",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -D runelite /app/bin/runelite",
+                "install -D RuneLite.jar /app/share/RuneLite.jar"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "runelite",
+                    "commands": [
+                        "ls -R /app | grep jagex",
```

Comment:
What is this good for? Seems to be just a check if the directory exist.

Reply:
Oh. I forgot to remove that.